bike-erp/src/redux/actions/OrderListActions/orderListAction.ts

```diff
@@ -0,0 +1,28 @@
+import {ADD_ITEM, REMOVE_ALL_ITEMS, REMOVE_ITEM} from '../../types/OrderListTypes/orderListTypes'
+
```

Comment:
add comments

Reply:
The redux action are self explanatory based on function names